### DISCUSSION TITLE
McAfee security alert - how to deal with situation

### DISCUSSION THREAD

**Discussion by petr-priborsky** [fluent/fluent-bit#8759] [General]
Hi,

we are using older version of fluent-bit installer for windows (https://releases.fluentbit.io/2.1/fluent-bit-2.1.9-win32.exe). we received an information, that some virus detection application reports an alert when installing (McAfee, Symantec). 
I checked situation with VirusTotal.com and it really identifies risks:

![image](https://github.com/fluent/fluent-bit/assets/147529537/7bd8ac7d-f3fb-4ab5-9204-be643dfb25d9)

Latest release 3.0.2 (https://releases.fluentbit.io/3.0/fluent-bit-3.0.2-win32.exe) has similar alert

![image](https://github.com/fluent/fluent-bit/assets/147529537/0b0e1d17-bc3f-4db7-a93b-fdcc618b186b)

How do you deal with this situation?

**Comment by agup006** [comment]
This might be best to ask the security vendors? If you could do that and report back that would be helpful to understand what we could change
